plugins/include/amxconst.inc

```diff
@@ -154,6 +155,18 @@ public stock const Float:NULL_VECTOR[3];
 #define CSW_VEST            31  // Custom
 #define CSW_VESTHELM        32  // Custom
 #define CSW_SHIELDGUN       99
+#define CSW_LAST_WEAPON     CSW_P90
+
+stock const CSW_ALL_WEAPONS      = (~(1<<CSW_VEST));
```

Comment:
Well, you have native like `get_user_weapon` in core which doesn't need cstrike be included and I thought that it's too late to move them, since some old plugins could fail to be recompiled.
We could `cstrike_const.inc` could be included in this file, so sure if it makes sense though.